recipe/recipe.yaml

```
@@ -0,0 +1,72 @@
+schema_version: 1
+
+context:
+  name: boost-histogram
+  version: 1.5.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/b/boost_histogram/boost_histogram-${{ version }}.tar.gz
+  sha256: 6e0f3dbdf7ab60ca1c61d5caa4f3f9e6097461feb65c6dbb01302fd88b1be524
+
+build:
+  number: 1
+  script:
+    - if: not (linux and ppc64le)
+      then: export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
+    - ${{ PYTHON }} -m pip install . -v
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then: python
+    - if: build_platform != target_platform
+      then: cross-python_${{ target_platform }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+    - cmake
+    - if: not win
+      then: make
+  host:
+    - python
+    - pip >=18
+    - scikit-build-core
+    - setuptools_scm
+  run:
+    - python
+    - numpy
+
+tests:
+  - python:
+      imports:
+        - boost_histogram
+  - files:
+      source:
+        - tests
```

```suggestion
        - tests/
```

Bug reported upstream: https://github.com/prefix-dev/rattler-build/issues/1480